doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -350,6 +350,36 @@ Matter
 
 The issues in this section are related to the :ref:`ug_matter` protocol.
 
+.. rst-class:: v2-6-1 v2-6-0 v2-5-3 v2-5-2 v2-5-1 v2-5-0
+
+KRKNWK-18916: Issues related to the  :kconfig:option:`CONFIG_CHIP_LAST_FABRIC_REMOVED_ERASE_AND_PAIRING_START` Kconfig option
+  When the Kconfig option is selected, there are two issues:
+
+  * An assert may occur after removing the last fabric because the OpenThread interface is still active despite the Thread stack being disabled.
+  * The device cannot be commissioned to the Matter network because the device's MAC address is the same as the one saved in the SRP server.
```

Comment:
I don't think it's about MAC address, right? IEEE 802.15.4 Extended address is not the same as MAC address. In  therms of Thread EUI64 is more something like the MAC address.

Reply:
Changed.